packages/frontend-2/lib/user/graphql/queries.ts

```diff
@@ -21,3 +21,13 @@ export const userEmailsQuery = graphql(`
     }
   }
 `)
+
+export const userActiveWorkspaceQuery = graphql(`
+  query UserActiveWorkspace {
+    activeUser {
```

Comment:
for both activeUser and activeWorkspace https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?source=copy_link#2851514fbf9a406da31e430a57859565